app/src/main/java/de/danoeh/antennapod/fragment/QueueFragment.java

```diff
@@ -521,6 +523,51 @@ private void reallyMoved(int from, int to) {
                     Log.d(TAG, "Write to database move(" + from + ", " + to + ")");
                     DBWriter.moveQueueItem(from, to, true);
                 }
+
+                @Override
+                public void onChildDraw(Canvas c,
+                                        RecyclerView recyclerView,
+                                        RecyclerView.ViewHolder viewHolder,
+                                        float displacementX,
+                                        float displacementY,
+                                        int actionState,
+                                        boolean isCurrentlyActive) {
+                    if (actionState == ItemTouchHelper.ACTION_STATE_SWIPE) {
+                        View itemView = viewHolder.itemView;
+
+                        Paint p = new Paint();
+                        p.setColor(getResources().getColor(R.color.accent_dark));
+                        if (displacementX > 0) {
+                            c.drawRect((float) itemView.getLeft(), (float) itemView.getTop(), displacementX,
+                                    (float) itemView.getBottom(), p);
+                        } else {
+                            c.drawRect((float) itemView.getRight() + displacementX, (float) itemView.getTop(),
+                                    (float) itemView.getRight(), (float) itemView.getBottom(), p);
+                        }
+
+                        super.onChildDraw(c, recyclerView, viewHolder,
+                                displacementX, displacementY,
+                                actionState, isCurrentlyActive);
+                    }
+                }
+
+                @Override
+                public float getSwipeEscapeVelocity(float defaultValue) {
+                    return defaultValue * 1.3f;
```

Comment:
pixels per second, this requires a much faster swipe  https://developer.android.com/reference/androidx/recyclerview/widget/ItemTouchHelper.Callback#getSwipeEscapeVelocity(float)

@keunes FYI